src/libre/print/dot.c

```diff
@@ -125,11 +125,11 @@ pp_iter(FILE *f, const struct fsm_options *opt,
 		break;
 
 	case AST_EXPR_FLAGS:
-		fprintf(f, "\tn%p [ label = <{FLAGS|{+", (void *) n);
+		fprintf(f, "\tn%p [ label = <FLAGS|{+", (void *) n);
 		re_flags_print(f, n->u.flags.pos);
-		fprintf(f, "|-");
+		fprintf(f, "}|{-");
 		re_flags_print(f, n->u.flags.neg);
-		fprintf(f, "> ];\n");
+		fprintf(f, "}> ];\n");
```

Comment:
Thanks! I have no idea what was going on here.

While you're here, could you remove the `sep` character for between flags please? So the output is just like `+imgs` etc.